### DISCUSSION TITLE
Sky LED do not work

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#2471] [Q&A] [answered]
Trying out the `leds-example.c` in Cooja, I noticed that Cooja and Z1 motes work as expected, but Sky does not. Is that expected (I'm still running Contiki-NG v4.7-dirty)?

I also noticed that only three of the mote types in Cooja menus can be compiled: cooja, sky, and Z1. All the rest of them say "The target platform "XXX" does not exist...".

**Comment by pjonsson** [comment, marked as answer]
The other platforms are not supported by Contiki-NG. A bunch of code for MSP430-based platforms has been removed from Cooja as well after 4.8.

There have been fixes to LEDs with Cooja after 4.8.

If LEDs are important to you, I suggest you try the develop branch before 4.9 is released and create issues for things that are not working as you expect.

**Reply by rahav** [reply]
Thank you for the quick response.
